pulp_puppet_extensions_admin/pulp_puppet/extensions/admin/structure.py

```diff
@@ -45,6 +34,7 @@
 
 # -- creation -----------------------------------------------------------------
 
+
```

Comment:
@dkliban Well in fact I did mean to. Because the editor was complaining about flake error(two blank lines before function or something). But I will remove it.